webseer_process.php

```diff
@@ -98,6 +98,17 @@
 	exit(1);
 }
 
+if (api_plugin_is_enabled('maint')) {
+	include_once($config['base_path'] . '/plugins/maint/functions.php');
+}
+
+if (function_exists('plugin_maint_check_cacti_host')) {
```

Comment:
Surely this should check for the correct function `plugin_maint_check_webseer_url` as it could be a version without it?

Reply:
I concur.  I incorrectly thought it was related to the API hook registration.